classes/core.php

```diff
@@ -1447,9 +1528,37 @@ static public function run_calculation($value, $field, $form)
 			return new WP_Error($field['ID'] . '-calculation',
 				__('Calculation is invalid (division by zero)', 'caldera-forms'));
 		}
+	
+		//If PHP version is less than 7.2.0, continue using old function
+		if(phpversion() < '7.2.0'){
```

Comment:
@New0 Please use version compare here. This code probably works as expected, but it's unreliable,  `version_compare` solves that.

https://www.php.net/manual/en/function.version-compare.php